extensions/wikia/AdEngine/AdEngine2ContextService.class.php

```diff
@@ -79,6 +79,7 @@ public function getContext( Title $title, $skinName ) {
 				] ),
 				'providers' => $this->filterOutEmptyItems( [
 					'evolve2' => $wg->AdDriverUseEvolve2,
+					'audienceNetwork' => $wg->AdDriverUseAudienceNetworkBidder,
```

Comment:
Oh, wait. It's being set by default to `false` in the backend, so I think there is no need to panic ;) https://github.com/Wikia/app/pull/13291/files#diff-013f1f772182745f48c5f1429d0cdd35R1254